karbon/karbon_cluster_registration_service_test.go

```diff
@@ -118,6 +152,168 @@ func TestKarbonCreateClusterRegistration(t *testing.T) {
 	// TODO get task uuid status
 }
 
+func TestKarbonCreateClusterRegistrationAndSetInfo(t *testing.T) {
+	interceptor := khttpclient.NewInterceptor(http.DefaultTransport)
+	creds := testhelpers.CredentialsFromEnvironment(t)
+	nkeClient, err := NewKarbonAPIClient(creds, WithRoundTripper(interceptor))
+	require.NoError(t, err)
+
+	kctx := mock.NewContext(mock.Config{
+		Mode: keploy.MODE_TEST,
+		Name: t.Name(),
+	})
+
+	test_cluster_name := strings.ToLower("cluster1")
+	test_cluster_uuid := strings.ToLower("634F5852-CE1B-465C-A95A-9B8DFFBDDE42")
+	test_category_mapping := map[string]string{
+		fmt.Sprintf("kubernetes-io-cluster-%s", test_cluster_name): "owned",
+		"KubernetesClusterName": test_cluster_name,
+		"KubernetesClusterUUID": test_cluster_uuid,
+	}
+	test_metadata_apiversion := "v1.1.0"
+	test_metadata := &Metadata{APIVersion: &test_metadata_apiversion}
+
+	t.Log("Get Cluster registration")
```

Comment:
Consider removing `t.Log`. Outside of a long-running e2e test, these don't really add much value to the test itself once the test has been written and pollute the signal instead (is test passing/failing).

Reply:
its debatable. as it can be used to debug in cases where asserts are missed.